mongo/collection.go

```diff
@@ -1573,9 +1561,7 @@ func (coll *Collection) FindOneAndReplace(ctx context.Context, filter interface{
 		}
 		op = op.Comment(comment)
 	}
-	if fo.MaxTime != nil {
-		op = op.MaxTimeMS(int64(*fo.MaxTime / time.Millisecond))
-	}
+	op = op.MaxTime(fo.MaxTime)
```

Comment:
Since we are no longer wrapping this condition in an if block, can we just extend the operation builder [here](https://github.com/mongodb/mongo-go-driver/blob/6e9fc900d992c6aa53d5004e6c75ba1bb994f2b4/mongo/collection.go#L1564) ?